Octokit/Octokit-Monotouch.csproj

```diff
@@ -351,6 +351,19 @@
     <Compile Include="Models\Response\ThreadSubscription.cs" />
     <Compile Include="Models\Request\MarkAsReadRequest.cs" />
     <Compile Include="Models\Request\NewThreadSubscription.cs" />
+    <Compile Include="Clients\IRepositoryContentsClient.cs" />
+    <Compile Include="Clients\RepositoryContentsClient.cs" />
+    <Compile Include="Models\Response\DirectoryContent.cs" />
+    <Compile Include="Models\Request\CreateFileRequest.cs" />
+    <Compile Include="Models\Request\Signature.cs" />
+    <Compile Include="Helpers\SerializeAsBase64Attribute.cs" />
+    <Compile Include="Models\Response\ContentType.cs" />
+    <Compile Include="Models\Response\RepositoryContent.cs" />
+    <Compile Include="Models\Response\RepositoryContentChangeSet.cs" />
+    <Compile Include="Models\Response\CommitContent.cs" />
+    <Compile Include="Models\Response\SignatureResponse.cs" />
+    <Compile Include="Helpers\PropertyOrField.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\iOS\Xamarin.MonoTouch.CSharp.targets" />
+  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
```

Comment:
Urgh, I need to renew my Xamarin license and ensure everything still builds there...